Scripts/Simulation_scripts/Energy_System.py

```diff
@@ -18,29 +18,51 @@
 import pandas as pd
 import datetime
 import math
+import os
 
 import sys
-sys.path.insert(0, '/***YOUR LOCAL FILE PATH***/CLOVER 4.0/Scripts/Generation scripts/')
+sys.path.insert(0, './Scripts/Generation_scripts/')
 from Solar import Solar
 from Diesel import Diesel
 import sys
-sys.path.insert(0, '/***YOUR LOCAL FILE PATH***/CLOVER 4.0/Scripts/Load scripts/')
+sys.path.insert(0, './Scripts/Load_scripts/')
 from Load import Load
 #%%
 class Energy_System():
-    def __init__(self):
-        self.location = 'Bahraich'
-        self.CLOVER_filepath = '/***YOUR LOCAL FILE PATH***/CLOVER 4.0'
+    def __init__(self,**kwargs):
+        # kwargs must include location
+        # kwargs can include energy_system_inputs (to override inputs in location files)
+        # kwargs can include scenario_inputs, location_inputs, device_inputs, diesel_inputs, grid_inputs, finance_inputs, GHG_inputs (passed on to other scripts to override inputs in location files)
+        self.kwargs = kwargs
+        self.location = kwargs.get('location')
+        self.CLOVER_filepath = '.'
```

Comment:
Here, the `os.getcwd()` would work well. Hard-coding in something like `.` shouldn't really be needed 😄